spk/ps3netsrv/src/service-setup.sh

```diff
@@ -7,6 +7,6 @@ export CFG_FILE PID_FILE SERVICE_PORT
 service_postinst ()
 {
     if [ "${SYNOPKG_PKG_STATUS}" == "INSTALL" ]; then
-        sed -i -e "s|@wizard_dir@|${wizard_data_volume}/${wizard_data_directory}|g" ${CFG_FILE}
+        sed -i -e "s|@wizard_dir@|${SHARE_PATH}|g" ${CFG_FILE}
```

Comment:
suggest that you must apply the changed placeholder in ps3netsrv.conf file

```suggestion
        sed -i -e "s|@share_path@|${SHARE_PATH}|g" ${CFG_FILE}
```

Reply:
Oh my bad, sorry.

Fixed and committed